apps/admin-x-activitypub/src/components/Activities.tsx

```diff
@@ -70,29 +91,79 @@ const getActorUrl = (activity: Activity): string | null => {
 
 const getActivityBadge = (activity: Activity): AvatarBadge => {
     switch (activity.type) {
+    case ACTVITY_TYPE.CREATE:
+        return 'user-fill'; // TODO: Change this
     case ACTVITY_TYPE.FOLLOW:
         return 'user-fill';
     case ACTVITY_TYPE.LIKE:
         if (activity.object) {
             return 'heart-fill';
         }
-    }    
-};
-
-const isFollower = (id: string, followerIds: string[]): boolean => {
```

Comment:
Moved this to be scoped inside of the component so we don't have to pass `followerIds`